js/templates/pageNav.html

```diff
@@ -8,7 +8,7 @@ <h3 class="margin0 marginTop25"><%= polyglot.t('Discover') %></h3>
   </a>
 </div>
 
-<div class="popMenu hide popMenu-navBar js-navProfileMenu">
+<div class="popMenu popMenu-navBar js-navProfileMenu">
```

Comment:
The problem is that it looks like the menu is not a consistent height, so any hard-coded value would be less than ideal. You could set it to the largest possible height, but then when it's smaller, there's what appears to be a slight delay in the animation since when the menu is off-screen (behind the navbar), it's not flush with the bottom of the navbar. That's minor though and we could probably live with it.

I'll explore seeing if we could offset it by 100% of the menu height. Top doesn't do that since it uses the container height to determine percentage rather than the menu height.

Anyways, I'll look into it.